extensions/plugin-basic-ui/src/basicUIPlugin.css.ts

```diff
@@ -32,6 +32,7 @@ const GLOBAL_VARS = {
   },
   modal: {
     borderRadius: "bottom-sheet-border-radius",
+    maxWidth: "max-width",
```

Comment:
Suggest the component-specific variable name for evading namespace conflict of future:
```suggestion
    maxWidth: "modal-max-width",
```

Reply:
Oh It's my mistake. Thank you!